tests/foreman/ui_airgun/test_smartclassparameter.py

```diff
@@ -33,16 +33,24 @@ def module_org():
     return entities.Organization().create()
 
 
+@fixture(scope='module')
+def module_loc():
+    return entities.Location(id=2).read()
+
+
 @fixture(scope='module')
 def content_view(module_org):
     return publish_puppet_module(
         PUPPET_MODULES, CUSTOM_PUPPET_REPO, module_org)
 
 
 @fixture(scope='module')
-def puppet_env(content_view):
-    return entities.Environment().search(
-        query={'search': u'content_view="{0}"'.format(content_view.name)})[0]
+def puppet_env(content_view, module_org):
+    env = entities.Environment().search(
+        query={'search': u'content_view="{0}" and organization_id={1}'.format(
+            content_view.name, module_org.id)}
+    )[0]
+    return env
```

Comment:
why setting single-use variable instead of just returning it?

Reply:
do not like to return directly function call results especially those ones that make external services calls with list access retrieval, but this is not a blocker for me if you insist I can change it :)